manifests/y/youtube-dl/youtube-dl/2021.12.17/youtube-dl.youtube-dl.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS0.5-1-22000-653
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: youtube-dl.youtube-dl
+PackageVersion: 2021.12.17
+MinimumOSVersion: 10.0.0.0
+InstallerType: portable
+Commands:
+- youtube-dl
+ReleaseDate: 2021-12-17
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/ytdl-org/youtube-dl/releases/download/2021.12.17/youtube-dl.exe
+  InstallerSha256: 26E5C00C35C5C3EDC86DFC0A720AED109A13B1B7C67AC654A0CE8FF82A1F2C16
+  AppsAndFeaturesEntries:
+  - DisplayName: youtube-dl
+    DisplayVersion: 2021.12.17
```

```suggestion
```
It is not recommended to have a display version which is the same as the package version

---

Could you elaborate? Something like this?
```suggestion
    DisplayVersion: 2021.12.17.0
```

In that case, the display version is equal to the file version of the executable.